data/web/inc/vars.inc.php

```diff
@@ -42,11 +42,13 @@
 $MAILCOW_APPS = array(
   array(
     'name' => 'SOGo',
-    'link' => '/SOGo/'
+    'link' => '/SOGo/',
+    'description' => 'SOGo is a collaborative software (groupware) server.'
```

Comment:
"SOGo is a web-based client for email, address book and calendar."
"Roundcube is a light-weight substitute of SOGo webmail."